lib/rubocop/cop/mixin/documentation_comment.rb

```diff
@@ -23,9 +23,9 @@ def documentation_comment?(node)
         end
       end
 
-      def preceding_comment?(n1, n2)
-        n1 && n2 && precede?(n2, n1) &&
-          comment_line?(n2.loc.expression.source)
```

Comment:
Note to self: These aren't always nodes. Rename to something that embodies that `n1` is either a `AST::Node` or a `Parser::Source::Comment`